setup.py

```diff
@@ -34,14 +34,21 @@
 )
 (
     is_release,
+    is_enterprise,
     version,
     version_major,
     version_minor,
     version_bug,
 ) = get_release_and_version(package_path)
 version_base = f"{version_major}.{version_minor}.0"
 
-_PACKAGE_NAME = "deepsparse" if is_release else "deepsparse-nightly"
+_PACKAGE_NAME = (
+    "deepsparse-ent"
+    if is_enterprise
+    else "deepsparse"
+    if is_release
+    else "deepsparse-nightly"
+)
```

Comment:
nit: In my opinion, this is difficult to read, could we refactor into a function call maybe and use normal `if ... elif .. else` construct, won't block the PR on this though